templates/_layouts/html.twig

```diff
@@ -60,6 +60,20 @@
           document.documentElement.className = document.documentElement.className.replace('has-no-js', 'has-js');
         </script>
         {% block head_end %}{% endblock %}
+
+        {# GA4 Google Analytics tracking #}
+        {% set googleId = '' %}
```

Comment:
Add a comment here with a `TODO:` prefix instructing us to fetch this value via from the authors. See other examples of `TODO:` comments in the boilerplate codebase.